ocfweb/about/templates/about/staff.html

```diff
@@ -82,13 +82,20 @@ <h3>Computer Lab &amp; Printing</h3>
     <div class="ocf-lab">
         <div class="container">
             <div class="row">
-                <div class="col-md-7">
-                    <img src="{% static 'img/about/server-lab.png' %}" alt="servers and lab" />
+                <div class="col-md-7 align-items-center">
+            <div class="col-md-7">
+                <br/>
+                <br/>
+            </div>
+                    <img src="{% static 'img/about/171-mlk-lab.jpg' %}" alt="OCF at MLK building, 2019 Spring" />
                 </div>
                 <div class="col-md-5">
                     <h2>We have a spiffy computer lab on southside.</h2>
                     <h3><a href="{% url 'register' %}">Create an account</a> and drop by!</h3>
                     <p>Our computer lab, located in <a href="{% url 'doc' 'services/lab' %}">171 MLK Student Union</a>, features high-res scanners, free printing, and high-end desktops with mechanical keyboards and GTX graphics cards, all running Debian GNU/Linux.</p>
+                    <div class="col-md-12">
```

Comment:
I don't think Bootstrap is intended to have nested columns. See if the `class="col-md-12"` is necessary.

Reply:
https://getbootstrap.com/docs/4.0/layout/grid/#nesting

nesting is a feature, and I think the alignment broke without it.

Reply:
You'll need to add a `<div class="row">` per those docs. Also, note that we are using Bootstrap 3, not 4.

Reply:
removed from the code